TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOpMain.java

```diff
@@ -75,15 +89,26 @@ public void initialize() {
         shooterSystem = new ShooterSystem(shot, telemetry, () -> pwrSelect);
         shootCommand = new Com_Shoot(shooterSystem);
         stopCommand = new Com_NoShoot(shooterSystem);
-
         shooterStart = (new GamepadButton(m_driverOp, GamepadKeys.Button.A))
                 .whenPressed(shootCommand);
         shooterStop = (new GamepadButton(m_driverOp, GamepadKeys.Button.B))
                 .whenPressed(stopCommand);
 
+        wobbleSystem = new WobbleSystem(pickup);
+        pickupCommand = new Com_PickUp(wobbleSystem);
+        goalLift = (new GamepadButton(m_driverOp, GamepadKeys.Button.Y))
+                .whenPressed(pickupCommand);
+
+        intakeSystem = new IntakeSystem(intake);
+        startIntakeCommand = new Com_IntakeStart(intakeSystem);
+        stopIntakeCommand = new Com_IntakeStop(intakeSystem);
+        toggleIntake = intakeSystem.intakeActive ? new GamepadButton(m_driverOp, GamepadKeys.Button.X)
+                .whenPressed(stopIntakeCommand) : new GamepadButton(m_driverOp, GamepadKeys.Button.X)
+                .whenPressed(startIntakeCommand);
```

Comment:
I don't think this works. I'd recommend using one singular definition for the button with a `toggleWhenPressed` binding. My suggestion is this:
```java
toggleIntake = new GamepadButton(m_driverOp, GamepadKeys.Button.X)
        .whenPressed(new ConditionalCommand(startIntakeCommand, stopIntakeCommand, intakeSystem::active))
        .toggleWhenPressed(new InstantCommand(intakeSystem::toggle));
```
Refer to my comment in IntakeSubsystem for more information.